### ISSUE TITLE
Poems containing “watch” have messed-up syllable counts

### ISSUE THREAD

**Issue by roryokane** [bilalq/Tranquillity-Editor#1]
Go to [tranquillpoet.com](http://tranquillpoet.com/) and enter this poem:

```
Does anyone have a watch with the correct time?
Hey there
Watch me fly
```

You’ll see that the syllable counts say this:

```
5function watch() { [native code] }1121
2
0function watch() { [native code] }11
```

But they should say this:

```
12
2
3
```

(This poem also demonstrates a separate bug where “anyone” counts as 2 syllables instead of 3 as it should. If you don’t expect that bug to be fixed yet, the first line should say “11” syllables, not “12”.)

**Comment by kotsoft** [comment]
There are some words whose syllable count can't be determined by algorithm alone. For those, TextStatistics.js allows us to keep a dictionary of problem words and their respective counts in line 142. I will issue a pull request where I've added the word "anyone". Possibly we should implement a queue where people can submit words they've noticed have the wrong count and we can review them?

**Comment by roryokane** [comment]
I just wrote up the “anyone” syllable count bug as a separate Issue, #3.

**Comment by kotsoft** [comment]
Could you give me the specs of your browser? I don't seem to be able to reproduce this on mine.

**Comment by roryokane** [comment]
I’m running Firefox 16.0.2 on Mac OS X. My User Agent string: `Mozilla/5.0 (Macintosh; Intel Mac OS X 10.6; rv:16.0) Gecko/20100101 Firefox/16.0`.

When I enter the poem into Google Chrome 23.0.1271.64 on the same computer the bug does not appear; the syllable counts are fine.

**Comment by bilalq** [comment]
Fixed by #12 because of this was caused by problems with the prototype chain.
